requirements-wmg-pipeline.txt

```diff
@@ -33,4 +34,5 @@ scipy==1.10.1
 SQLAlchemy==1.4.49
 SQLAlchemy-Utils==0.41.1
 tenacity==8.2.2
-tiledb==0.21.4  # Portal's tiledb version should always be the same or older than Explorer's
\ No newline at end of file
+tiledb==0.22.3  # Portal's tiledb version should always be the same or older than Explorer's
+tiledbsoma==1.4.3  
```

Comment:
tiledb and tiledbomsa are transitive dependencies of cellxgene-census; any reason to explicitly include them?

Also, I think the Explorer-related comment doesn't apply to the requirements for the wmg pipeline docker image--this is only matters for the dataset pipeline, which produces the cxg files; could remove the comment if leaving in the explicit tiledb dependency